### DISCUSSION TITLE
Scrubbing during replay viewing is too slow

### DISCUSSION THREAD

**Discussion by 25pwn** [ppy/osu#20229] [Q&A] [answered]
The transition for scrubbing during replay viewing is too slow. For example, when scrubbing to the end, [Tsukinami[Nostalgia]](https://osu.ppy.sh/beatmapsets/896080#osu/1872396)(4:37) takes over 5 seconds, and [PLUM MEGAMIX[PLUM DIMENSIONS]](https://osu.ppy.sh/beatmapsets/1463653#osu/3006446)(9:54) takes over 16.

**Comment by peppy** [comment, marked as answer]
It's limited by game performance / frame rate. It'll improve as we improve performance and decouple seeking further from frame rate.
